app/scripts/controllers/build.js

```diff
@@ -94,15 +110,19 @@ angular.module('openshiftConsole')
       $scope.alerts["load"] = {
         type: "error",
         message: "The build details could not be loaded.",
-        details: "Reason: " + $filter('getErrorDetails')(e)
+        details: "Reason: " + $filter("getErrorDetails")(e)
       };
     };
 
     var buildConfigResolved = function(buildConfig, action) {
       if (action === "DELETED") {
         $scope.alerts["deleted"] = {
           type: "warning",
-          message: "Build configuration " + $scope.buildConfigName + " has been deleted."
+          message: (
+            "Build configuration " +
```

Comment:
We don't use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in the console yet to avoid a transpiler, but we do use them in the tests since they run in node.